Gemfile

```diff
@@ -134,6 +133,7 @@ end
 group :production, :test do
   # Puma will be our app server
   gem 'puma'
+  gem 'aws-sdk-s3'
```

Comment:
Are our tests uploading uploads to S3? If not, I think we can move it down to the `:production` group.

Reply:
Our ci is using it to upload screenshots of failed tests to s3. Moved it to ci instead of test